pelican/settings.py

```diff
@@ -139,7 +140,12 @@ def load_source(name, path):
     'TEMPLATE_PAGES': {},
     'TEMPLATE_EXTENSIONS': ['.html'],
     'IGNORE_FILES': ['.#*'],
-    'SLUG_SUBSTITUTIONS': (),
+    'SLUG_REGEX_SUBSTITUTIONS': [
+        (r'[^\w\s-]', ''),
+        (r'(?u)\A\s*', ''),
+        (r'(?u)\s*\Z', ''),
+        (r'[-\s]+', '-'),
+    ],
```

Comment:
Since this is now exposed, it would be nice to add a comment for each line to describe what it's doing. Same with the docs.

Reply:
Yes, good idea